jetty-ee10/jetty-ee10-servlet/src/test/resources/jetty-logging.properties

```diff
@@ -1,5 +1,5 @@
 #org.eclipse.jetty.LEVEL=DEBUG
-#org.eclipse.jetty.http.LEVEL=DEBUG
+org.eclipse.jetty.http.LEVEL=DEBUG
```

Comment:
Probably shouldn't commit this change - just to reduce the amount of output from tests.